src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoMend.kt

```diff
@@ -105,12 +118,38 @@ internal object AutoMend : Module(
                 if (autoThrow && player.heldItemMainhand.item === Items.EXPERIENCE_BOTTLE) {
                     val packet = PlayerPacket(rotating = true, rotation = Vec2f(player.rotationYaw, 90.0f))
                     PlayerPacketManager.addPacket(AutoMend, packet)
+                    isMending = true
                     if (validServerSideRotation() && throwDelayTimer.tick(throwDelay.value.toLong())) {
                         playerController.processRightClick(player, world, EnumHand.MAIN_HAND)
                     }
                 }
             }
         }
+
+        safeListener<TickEvent.ClientTickEvent> {
+            if(!pauseAutoArmor) return@safeListener
```

Comment:
Style, there should be a space before the open bracket

Reply:
is there a reason why we are adding another ClientTickEvent here? 

Also i think we need a module pause framework instead of hacking at specific modules to pause them individually. This would be outside the scope of this PR but a good enhancement.

Reply:
That's what I was thinking with my comment on line 71